src/backend/joanie/settings.py

```diff
@@ -420,6 +420,11 @@ class Base(Configuration):
             environ_name="JOANIE_PAYMENT_CONFIGURATION", environ_prefix=None
         ),
     }
+    PAYMENT_SCHEDULE_LIMITS = values.DictValue(
```

Comment:
Describe more your commit please. What is this setting ? What it is doing. For a commit as simple as this one you should describe it more